docs/benchmarks/data-analytics.md

```diff
@@ -37,7 +37,7 @@ $ docker run -d --net host --name slave02 cloudsuite/hadoop:2.9.2 slave $IP_ADRE
 
 ...
 ```
-Note : Start each slave on a different VM.
+Note : Start each slave on a different VM. If they are running on the same host rather than multiple VMs, you should set `IP_ADRESS_MASTER` to `127.0.1.1`.
```

Comment:
Do you mean `127.0.0.1`?

Reply:
I saw what you wrote in the PR message but I was still not sure about this.
Can't we just use `localhost`?

Reply:
I guess the master is listening on 127.0.1.1 (the localhost 127.0.0.1), so 127.0.0.1 doesn't work. (Just a guess, because I saw in the entrypoint.sh they change the mapping of `127.0.1.1` rather than `127.0.0.1`.

Reply:
Hmmm, this is strange. I would say the config should set the IP to `0.0.0.0` to make the container listen on all the interfaces. Take a look at [this file](https://github.com/big-data-europe/docker-hadoop/blob/master/base/entrypoint.sh) and check whether we can get anything useful out of it.

Reply:
If you look at their configuration file, they actually insert the `0.0.0.0` in the hdfs-site.xml file. 

```
    addProperty /etc/hadoop/hdfs-site.xml dfs.namenode.rpc-bind-host 0.0.0.0
    addProperty /etc/hadoop/hdfs-site.xml dfs.namenode.servicerpc-bind-host 0.0.0.0
    addProperty /etc/hadoop/hdfs-site.xml dfs.namenode.http-bind-host 0.0.0.0
    addProperty /etc/hadoop/hdfs-site.xml dfs.namenode.https-bind-host 0.0.0.0
    addProperty /etc/hadoop/hdfs-site.xml dfs.client.use.datanode.hostname true
    addProperty /etc/hadoop/hdfs-site.xml dfs.datanode.use.datanode.hostname true
```
However, we don't have these information in our hdfs-site.xml.

Reply:
Right. For now it's fine but I believe it would be good to come back to this later and try to make it work on all the interfaces and accessible from outside. Please create an issue for this so we come back to it later when we are done with everything.